submissions/israelOladosu/database/transportationQuery.sql

```diff
@@ -0,0 +1,47 @@
+-- Total number of passengers who suvrvived
+SELECT COUNT(survived) FROM accidents WHERE survived = 1;
+-- answer: 270
```

Comment:
I got a different answer when I ran it. Kindly confirm if this is the right result

Reply:
I started the whole process again, dropped the database and recreated it with the columns and then imported the data all over again.

Only this changed to 269, others where the same